# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/encoding/legacy-mb-korean/euc-kr/euckr-decode-ksc_5601.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/encoding/legacy-mb-korean/euc-kr/euckr-decode-ksc_5601.html.ini
--- a/testing/web-platform/meta/encoding/legacy-mb-korean/euc-kr/euckr-decode-ksc_5601.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/encoding/legacy-mb-korean/euc-kr/euckr-decode-ksc_5601.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -2,7 +2,7 @@
 
 [euckr-decode-ksc_5601.html?16001-17000]
   expected:
-    if webrender and debug and (os == "linux") and not fission: ["OK", "CRASH"]
+    if (os == "linux") and debug and webrender and not fission: ["OK", "CRASH"]
 
 [euckr-decode-ksc_5601.html?2001-3000]
 